aws/sdk/integration-tests/s3/tests/express.rs

```diff
@@ -0,0 +1,249 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+use std::time::{Duration, SystemTime};
+
+use aws_config::Region;
+use aws_sdk_s3::presigning::PresigningConfig;
+use aws_sdk_s3::primitives::SdkBody;
+use aws_sdk_s3::{Client, Config};
+use aws_smithy_runtime::client::http::test_util::dvr::ReplayingClient;
+use aws_smithy_runtime::client::http::test_util::{
+    capture_request, ReplayEvent, StaticReplayClient,
+};
+use aws_smithy_runtime::test_util::capture_test_logs::capture_test_logs;
+use http::Uri;
+
+#[tokio::test]
+async fn list_objects_v2() {
+    let _logs = capture_test_logs();
+
+    let http_client =
+        ReplayingClient::from_file("tests/data/express/list-objects-v2.json").unwrap();
+    let config = aws_config::from_env()
+        .http_client(http_client.clone())
+        .no_credentials()
```

Comment:
if this has no creds, how does it work?

Reply:
A subsequent call to `.with_test_defaults()` will add one.